scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1351,8 +1351,8 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
     Parameters
     ----------
     A : An N x N matrix, array, sparse matrix, or LinearOperator representing
```

Comment:
This line actually should read 
```
    A : ndarray, sparse matrix or LinearOperator
        A square operator representing the operation `A * x`, where `A` is 
        real symmetric or complex hermitian. For buckling mode (see below) 
        `A` must additionally be positive-definite.
```
Otherwise it is trying to cast the sentence as a type annotation.